mpt_trie/src/partial_trie.rs

```diff
@@ -211,6 +215,12 @@ impl PartialTrie for StandardTrie {
         Self(n)
     }
 
+    // TODO(Robin): Do we ever want to use `StandardTrie` within the other crates?
+    // In which case we may want to include the strategy as part of the struct.
+    fn new_with_strategy(n: Node<Self>, _collapse_strategy: OnOrphanedHashNode) -> Self {
+        Self(n)
+    }
+
```

Comment:
nonblocking/nit: `StandardTrie` doesn't have hash nodes, so doesn't really make sense here.

In general I think we can split the two trie types up more, but that should be future work.

Reply:
@0xaatif Oh yeah so just for clarification, `StandardTrie` _does_ have `hash` nodes. The only difference between `StandardTrie` & `HashedPartialTrie` is `StandardTrie` does not cache the trie roots of a node and its children.